src/locksManager.js

```diff
@@ -0,0 +1,71 @@
+/* exported LocksManager */
+
+const {Gio, GLib, GObject} = imports.gi;
+
+const BusName = 'com.endlessm.GameStateService';
+const BusPath = '/com/endlessm/GameStateService';
+const BusIface = `
+    <node>
+      <interface name='com.endlessm.GameStateService'>
+        <method name='Get'>
+          <arg type='s' name='key' direction='in'/>
+          <arg type='v' name='value' direction='out'/>
+        </method>
+        <method name='Set'>
+          <arg type='s' name='key' direction='in'/>
+          <arg type='v' name='value' direction='in'/>
+        </method>
+      </interface>
+    </node>
+`;
+
+var LocksManager = GObject.registerClass({
+    Signals: {
+        changed: {},
+    },
+}, class LocksManager extends GObject.Object {
+    _init(props = {}) {
+        super._init(props);
+        const Proxy = Gio.DBusProxy.makeProxyWrapper(BusIface);
+        this._proxy = new Proxy(Gio.DBus.session, BusName, BusPath);
+        this._proxy.connect('g-signal', this._onChanged.bind(this));
```

Comment:
How can this do anything, given that the DBus interface does not have any signals in it?